instrumentation/kamon-elasticsearch/src/main/java/kamon/instrumentation/elasticsearch/highlevel/SyncClientInstrumentation.java

```diff
@@ -0,0 +1,18 @@
+package kamon.instrumentation.elasticsearch.highlevel;
+
+import kanela.agent.libs.net.bytebuddy.asm.Advice;
+
+public class SyncClientInstrumentation {
```

Comment:
Is there any particular reason why `AsyncClientInstrumentation` and `SyncClientInstrumentation` aren't just the same advice? Also, a more specific name would help :)

Reply:
D'oh, not reason. 
Will merge them with a prettier name.